src/NavigationBar/Title.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { View, Image, Text } from 'react-native'
+
+import titlePlaceholder from './nav-title-image-placeholder.png'
+
+export const Title = ({ titleOptions, variant }) => {
+  const {
+    enabled,
+    universalLayout,
+    universalUseLogo,
+    universalLogoImage,
+    universalText,
+    mobileUseLogo,
+    mobileText,
+    mobileLogo,
+    mobileAlignment,
+    desktopUseLogo,
+    desktopLogo,
+    desktopText,
+    styles,
+  } = titleOptions
+  if (!enabled) {
+    return <View />
+  }
+
+  if (variant === 'desktop') {
+    const useLogo = universalLayout ? universalUseLogo : desktopUseLogo
+    const logo = universalLayout ? universalLogoImage : desktopLogo
+    const text = universalLayout ? universalText : desktopText
+
+    if (useLogo) {
+      return (
```

Comment:
Just a comment for the future
Is there a way we can get these if content into their own components or in a way where this section is not that long, the useLogo logic seems repetitive as well